src/middlewared/middlewared/etc_files/ctld.py

```diff
@@ -212,7 +213,7 @@ def main(middleware):
     poolthreshold = {}
     zpoollist = {i['name']: i for i in middleware.call_sync('zfs.pool.query')}
 
-    system_disks = middleware.call_sync('device.get_disks')
```

Comment:
Just to confirm, is this not providing all the disks ?

Reply:
It is but not really what the question is in reference to?

Reply:
Oh I see, you're talking about sharing `multipath` raw disks

Reply:
Hmm, no just kidding. Not sure what you're asking? 😄